src/image/PhotoCalib.cc

```diff
@@ -465,38 +544,68 @@ double PhotoCalib::evaluate(lsst::geom::Point<double, 2> const &point) const {
         return _calibration->evaluate(point);
 }
 
+ndarray::Array<double, 1> PhotoCalib::evaluateArray(ndarray::Array<double, 1> xx,
+                                                    ndarray::Array<double, 1> yy) const {
```

Comment:
Convention is to pass arrays by const reference: `ndarray::Array<double, 1> const & xx`.

It's not a big deal either (neither is our opposite convention of passing `shared_ptr` by value), but best to be consistent.